master/reference/advanced/etcd-rbac/users-and-roles.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Creating Users and Roles in etcd
+---
+
+## Overview
+
+RBAC is *Role* Based Access Control so roles must be setup and also users
+that are given those roles must be created.  This document helps with the
+creation of both of those.
+
+> **Note:** The etcd release 3.x+ supports both v2 and v3 of its API.
+The etcd server keeps the roles and users separate, this means that if a
+user/role is created with the v2 API it will not appear in the v3 API.
+When adding roles and users they must be added through the API version that
+matches the version the component will be using.  This concern can be ignored
+if all roles and users are added through both API versions.
+
+## Users and Roles creation guides
+
+Use the following guides to setup your users, roles, and assignment of roles
```

Comment:
I think this comment is applying to `you must pass a proper CA and cert/key pair to the commands used in the below guides`.  I don't know that it should be under a pre-req section, it would be like adding `you must use the etcdctl tool to create users and roles`.  I was including it more as a reminder that the user would need to do that.  Would it be better if I changed `you must pass` to `you will need to pass`?